src/stateAggregator/accessors/orgAccessor.ts

```diff
@@ -61,10 +62,14 @@ export abstract class BaseOrgAccessor<T extends ConfigFile, P extends ConfigCont
     this.files = this.getAllFiles();
   }
 
-  public async read(username: string, decrypt = false, throwOnNotFound = false): Promise<P> {
-    const config = await this.initAuthFile(username, throwOnNotFound);
-    this.configs.set(username, config);
-    return this.get(username, decrypt) as P;
+  public async read(username: string, decrypt = false, throwOnNotFound = true): Promise<P> {
+    try {
+      const config = await this.initAuthFile(username, throwOnNotFound);
+      this.configs.set(username, config);
+      return this.get(username, decrypt) as P;
+    } catch (err) {
+      return {} as P;
```

Comment:
`{}` is not really `P` 

return type for `read` could be `Promise<P | Record<string,never>>` so that a consumer knows they can't rely on the required properties of P being present in the result of `read`

[an overload where throwOnNotFound = true could safely have output signature Promise<P>]

I'd prefer returning `undefined` (and have that be part of the return type) than an empty object.